browser/src/layer/tile/SheetGeometry.ts

```diff
@@ -100,19 +102,34 @@ export class SheetGeometry {
 		// We will check one special case here: Only a few rows/columns are visible on top, others are hidden all the way to the bottom/right.
 		const hiddenInfo = column ? sheetGeomJSON.columns.hidden.trim() : sheetGeomJSON.rows.hidden.trim();
 
-		// Check if our special case may exist.
-		if (hiddenInfo.split(' ').length === 2) {
-			const parts = hiddenInfo.split(' ');
-			if (parseInt(parts[1]) === 1048575 || parseInt(parts[1]) === 16383) {
-				return parseInt(parts[0].split(':')[1]);
-			}
+		const isFirstHidden = hiddenInfo.indexOf('1') === 0;
```

Comment:
can we put comment here with the format, example of hiddenInfo?
because it's hard to know what should be inside

Reply:
Yeah, i needed to read the core side code :) I already explained below a bit but let me elaborate it more.